src/python/WMCore/Storage/FileManager.py

```diff
@@ -7,22 +7,16 @@
 
 """
 
-import os
 import logging
-log = logging
+import time
 import traceback
-from WMCore.WMException import WMException
 
-from WMCore.Storage.StageOutError import StageOutError
-from WMCore.Storage.StageOutError import StageOutFailure
-from WMCore.Storage.StageOutError import StageOutInitError
-from WMCore.Storage.DeleteMgr import DeleteMgr
-from WMCore.Storage.Registry import retrieveStageOutImpl, RegistryError
+from WMCore.Storage.Registry import RegistryError, retrieveStageOutImpl
 from WMCore.Storage.SiteLocalConfig import loadSiteLocalConfig
+from WMCore.Storage.StageOutError import StageOutError, StageOutFailure
+
+log = logging
 
-import WMCore.Storage.Backends
-import WMCore.Storage.Plugins
```

Comment:
We need to keep these 2 imports otherwise we don't get the plugins. Since I made this mistake already, how about you add a comment above to avoid accidental deletions?

Reply:
OK, @amaltaro this is fixed. Thanks!